search/views.py

```diff
@@ -0,0 +1,31 @@
+"""View for search"""
+from rest_framework.response import Response
+from rest_framework.status import HTTP_405_METHOD_NOT_ALLOWED
+from rest_framework.views import APIView
+
+from channels.constants import (
+    COMMENT_TYPE,
+    POST_TYPE,
+)
+from open_discussions import features
+from search.api import execute_search
+
+
+VALID_CONTENT_TYPES = ('', COMMENT_TYPE, POST_TYPE)
+VALID_SORT_OPTIONS = ('', 'score', 'num_comments', 'created')
```

Comment:
Are these for use in the future or did you mean to add some validation code for this?

Reply:
I think that's just a copy/paste error, I'll remove it

Reply:
Actually this was a work in progress for validation, but we don't need it here if we're constructing the query on the frontend